ui/src/components/ConfigurationForm/ConfigurationForm.component.tsx

```diff
@@ -119,13 +165,53 @@ export const ConfigurationForm: React.FC<ConfigurationFormProps> = ({ isEmbedded
     });
   };
 
+  const [snackbar, setSnackbar] = useState<SnackbarConfig>({
+    open: false,
+    severity: 'info',
+  });
+  const handleSnackClose = (event?: SyntheticEvent | Event, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    setSnackbar({ open: false });
```

Comment:
Nit: this sets severity to `undefined` and `undefined` has color green. So when it fades, it flickers for a miliseconds from blue to green. Doing `setSnackbar({...snackbar, open: false})` should prevent it.

Reply:
Good catch! I haven't noticed that. Fixed.